src/main.c

```diff
@@ -109,6 +110,7 @@ void AgbMain()
     ClearDma3Requests();
     ResetBgs();
     SetDefaultFontsPointer();
+    MgbaOpen();
```

Comment:
Suggestion: Add this under an ifdef/config option. Not everybody might want it enabled as default and there is no reason to force it on -afail-